### DISCUSSION TITLE
코드하이라이트 적용 시 배경색이 적용되지 않음

### DISCUSSION THREAD

**Discussion by Scincy** [pronist/hELLO#135] [Ideas]
안녕하세요. 만들어주신 스킨 정말 잘 사용하고 있습니다.
개발 포폴용으로 조금씩 글을 쌓아 가려고 블로그 설정 중인데요, 코드 하이라이팅을 적용 했을 때 코드가 하이라이팅 되어야 할 부분이 글 색만 적용되고 배경 색은 적용되지 않는 문제를 겪고 있습니다. (다크모드, 라이트 모드 둘 다)
티스토리의 코드하이라이팅 플러그인이 설치되어 있었을 때 hELLO 스킨의 코드하이라이트 테마를 바꿨었고,
나중에 티스토리 코드 하이라이트 플러그인이 같이 설치되어 있으면 적용되지 않는다는 글을 읽고나서 티스토리 코드하이라이팅 플러그인을 삭제 했었는데.. 혹시 이게 원인일까요?
제가 무언가 실수로 하지 않은 작업이 있다면 알려 주시면 감사하겠습니다.

설정한 테마
* 다크모드 : github-dark
* 라이트모드 : github-dark-dimmed

사용환경 : MS Edge
블로그 포스트 : https://scincy.tistory.com/8
![image](https://user-images.githubusercontent.com/25105149/194982979-e0374f76-2f81-4149-8c48-6795d570746f.png)

**Comment by pronist** [comment]
@Scincy 안녕하세요. 

문제를 확인하려고 했으나 현재 카카오톡을 비롯한 티스토리 서버가 먹통이 되어서 확인을 할 수 없는 상황이군요 ㅠㅠ
서버가 복구되면 문제를 확인하도록 하겠습니다. :)

**Reply by Scincy** [reply]
티스토리 이제 정상적으로 접속 되네요~

**Comment by pronist** [comment]
@Scincy 안녕하세요. 늦었지만 확인해보았습니다.

확인결과, 현재 제 스킨의 기본 스타일에서 코드블록의 라이트와 다크테마에 대해 배경 설정을 강제화 하고 있는 것때문에 스타일이 적용되지 않는 것으로 파악했습니다. 이 부분은 다음 패치에 수정해야 할 듯 합니다. 감사합니다. :)
